setup.py

```diff
@@ -32,10 +32,10 @@ def read(fname):
     # For python2.6 we have to require argparse since it
     # was not in stdlib until 2.7.
     requires.append('argparse>=1.4')
-    requires.append('paramiko==2.3.2')
+    requires.append('paramiko==2.3.3')
     requires.append('pycparser==2.18')
 else:
-    requires.append('paramiko>=2.3.2')
+    requires.append('paramiko>=2.3.3')
```

Comment:
Could this install 2.4.1?

https://nvd.nist.gov/vuln/detail/CVE-2018-1000805

Also, trying out a new github feature.
```suggestion
    requires.append('paramiko==2.3.3')
```